gradle.properties

```diff
@@ -1,4 +1,5 @@
-voxelsniper.version=8.12.1
+voxelsniper.version=8.12.2
+voxelsniper.forge-version=1.20.1
```

Comment:
I'd say that `voxelsniper.forge.version` would be more semantic

Reply:
Also, `forge.mc-version` would be even better, since forge-version can also mean the `47.5.4.3` number

Reply:
good point, will change that

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/209/commits/8e3c43c8dde648d3b09bcd347e74b9e82764762e